nightguard WatchKit Extension/AppMessageService.swift

```diff
@@ -35,6 +35,21 @@ class AppMessageService : NSObject, WCSessionDelegate {
         }
     }
     
+    func keepAwakePhoneApp() {
+        
+        // send a dummy message to keep the phone app awake (start the app if is not started)
```

Comment:
This still feels like a hack, right? It's really a shame how hard it is to keep an app running. Even with background updates running. If I where Apple, I would suggest a special App-Type "Medical" with more rights to consume battery :-/ . But it seems like the running app industry has a better lobby to get such extra rights...

Reply:
Yeah, you're right! It's a hack, but I noticed that the phone app is alive all the time after doing it.
Very true about Apple policies. It seems that if you have a name, like Dexcom, you can obtain all the 288 updates per day. Open source is not loved, instead.. I don't know how easy is to obtain Apple approval if you add HealthKit capabilities to your app. By the way, is there any chance for implementing the background alarms using HealthKit? Or you should start running first, and only after you'll get the haptic feedback working? :)))